applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAX.java

```diff
@@ -345,6 +347,129 @@ public void setTestDeviceAdvertisingIds(final List<String> rawAdvertisingIds)
         }
     }
 
+    // Data Passing
+
+    public void setTargetingDataYearOfBirth(final int yearOfBirth)
+    {
+        if ( sdk == null )
+        {
+            logUninitializedAccessError( "setTargetingDataYearOfBirth" );
+            return;
```

Comment:
Should we add support for setting targeting data before init, like we do with verbose logging?

Reply:
That's a good point, @JonathanLiuApp since you added that recently for Unity, could you link your commit here for @alhiwatan to reference?

Reply:
Here is the PR: https://github.com/AppLovin/Unity-Plugin/pull/374

Reply:
Thanks for pointing out this and the PR.